client/ayon_blender/api/pipeline.py

```diff
@@ -488,12 +496,17 @@ def metadata_update(node: bpy.types.bpy_struct_meta_idprop, data: Dict):
     Existing metadata will be updated.
     """
 
-    if not node.get(AVALON_PROPERTY):
-        node[AVALON_PROPERTY] = dict()
+    if not node.get(AYON_PROPERTY) and not node.get(AVALON_PROPERTY):
+        node[AYON_PROPERTY] = dict()
+
+    ayon_property_name = (
+        AYON_PROPERTY if not node.get(AVALON_PROPERTY)
+        else AVALON_PROPERTY
+    )
```

Comment:
This should be reversed?

If AYON Property is present, it should be used.. it should not use the OLDER key if BOTH were present, right?